addon/mirage/factory.js

```diff
@@ -1,3 +1,21 @@
+import Ember from 'ember';
+
+function generateValue(type, key) {
+  if (type === 'string') {
+    return (i) => `${key} #${i}`;
```

Comment:
'arrow function syntax (=>)' is only available in ES6 (use esnext option).<br>Unexpected '`'.<br>'i' is defined but never used.